packages/i18next/package.json

```
@@ -15,19 +15,12 @@
 		"./register": {
 			"import": "./register.mjs",
 			"require": "./register.js"
-		},
-		"./register-discordjs": {
-			"import": "./register-discordjs.mjs",
-			"require": "./register-discordjs.js"
 		}
 	},
 	"sideEffects": [
 		"./dist/register.js",
 		"./register.js",
-		"./register.mjs",
-		"./dist/register-discordjs.js",
-		"./register-discordjs.js",
-		"./register-discordjs.mjs"
+		"./register.mjs"
```

```suggestion
		"./dist/register.js",
		"./dist/register.mjs",
		"./register.mjs"
```

Uh this was never right to begin with lol, oops